config/aes_sources.ini

```diff
@@ -30,8 +30,3 @@ name = UNESCO Atlas of the World's Languages in Danger
 reference_id = hh:hel:Moseley:Atlas:2010
 url = http://www.unesco.org/languages-atlas/
 
-[SafarWebster]
```

Comment:
I added this AES source recently, for one of the sign-languages. Is the removal intentional?